### PR TITLE
Cookies: Use different DB for different accounts

### PR DESCRIPTION
**Pull request by guruz**
This is a follow up to #5469

### CODE REVIEW

File: src/libsync/account.h [hunk 1]
```diff
@@ -172,6 +172,7 @@ class OWNCLOUDSYNC_EXPORT Account : public QObject {
 
     void clearCookieJar();
     void lendCookieJarTo(QNetworkAccessManager *guest);
+    QString cookieJarPath();
```
Comment (ogoffart) [inline src/libsync/account.h]: Please make it private.

**Comment by SamuAlfageme** [comment]
Minor detail: there's no migration strategy for `cookies.db` to be split into different `cookiesN.db`: the old cookies file is not removed nor its contents distributed.

File: src/libsync/account.h (hunk 1 above)
Comment (guruz) [inline src/libsync/account.h]: It's accessed from 2 different files/classes which are not the account, can't do it without friending both.

**Comment by guruz** [comment]
> Minor detail: there's no migration strategy for cookies.db to be split into different cookiesN.db: the old cookies file is not removed nor its contents distributed.

Hm, I just assumed it's fine because the behaviour should be similar to as if the cookies had been forgotten/expired.

**Pull request merged**